data/meson.build

```diff
@@ -0,0 +1,8 @@
+i18n.merge_file(
+    input: 'privacy.appdata.xml.in',
+    output: 'io.elementary.wingpanel.privacy.appdata.xml',
+    po_dir: join_paths(meson.source_root(), 'po', 'extra'),
```

Comment:
Don't forget to create meson.build, LINGUAS, and POTFILES in `po/extra` as you set up here